.grabl/automation.yml

```diff
@@ -290,13 +198,174 @@ build:
       monitor: |
         sleep 20s
         tail -f -n +1 ./dist/typedb-cluster-all-linux/server/logs/typedb.log
-    test-performance-typedb-cluster-three-benchmark:
+    test-performance-typedb-cluster-one-benchmark:
       machine: 16-core-32-gb
-      image: vaticle-ubuntu-21.04
-      dependencies: [test-performance-typedb-cluster-three-server-1, test-performance-typedb-cluster-three-server-2, test-performance-typedb-cluster-three-server-3]
+      image: vaticle-ubuntu-20.04
+      dependencies: [test-performance-typedb-cluster-one-server]
       timeout: "30m"
       command: |
        bazel run //:benchmark -- \
          --database typedb-cluster \
          --address $GRABL_EXPORT_PERFORMANCE_TYPEDB_CLUSTER_SERVER_URI \
-         --config /home/grabl/$GRABL_REPO/config/simulation.yml
+         --config /home/grabl/$GRABL_REPO/config/simulation.yml \
+#         --factory $GRABL_TRACING_URI \
+#         --org $GRABL_OWNER \
+#         --repo $GRABL_REPO \
+#         --commit $GRABL_COMMIT \
+#         --username $GRABL_OWNER \
+#         --token $GRABL_TOKEN
+#    test-performance-typedb-cluster-three-bootstrapper:
```

Comment:
we comment out the the 3-node cluster tests for now, and keep only Neo4j, TypeDB, and TypeDB Cluster with 1 node as our baseline